File: src/android/framework/LineSegment.java
//-------------------------------------------------------------------------------------------------------
// Copyright (C) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
//-------------------------------------------------------------------------------------------------------
package Windows.UI.Xaml.Controls;

import android.graphics.PointF;

public class LineSegment extends PathSegment {
    PointF _point;

    public PointF getPoint() {
        return _point;
    }
    public void setPoint(PointF point) {
        _point = point;
    }
}

File: www/framework/LineSegment.js
//-------------------------------------------------------------------------------------------------------
// Copyright (C) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
//-------------------------------------------------------------------------------------------------------

//
// A line between two points.
//
function LineSegment() {
    ace.UIElement.call(this, "Windows.UI.Xaml.Media.LineSegment");
};

// Inheritance
LineSegment.prototype = Object.create(ace.PathSegment.prototype);

// The end point of the line segment
LineSegment.prototype.getPoint = function () { return this.get("LineSegment.Point"); };
LineSegment.prototype.setPoint = function (point) { this.set("LineSegment.Point", point); };

module.exports = LineSegment;
